local/tests/ci_tests.bash

```diff
@@ -14,13 +14,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-IMAGE=gcr.io/clusterfuzz-images/ci
+IMAGE=gcr.io/clusterfuzz-images/ci:bc696af-202407021946
```

Comment:
should this be vguidi?

Reply:
I built this like, 1.5mo ago, but this is indeed python311.
Can change it for consistency and readability to point to the vguidi tag